src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(String color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    public Circle(String color, String name, int radius) {
+        super(color, name);
+        this.radius = radius;
+    }
+
```

Comment:
getArea() is a method from 'AreaCalculator' interface. It should be implemented here. Fix it in other figures
```suggestion
    @Override
    public double getArea() {
        return radius * radius * Math.PI;
    }

```